lib/omnibus/logger.rb

```diff
@@ -135,7 +135,7 @@ def format_message(severity, progname, message)
       else
         left = "#{format_severity(severity)} | "
       end
-      "#{left.rjust(LEFT)}#{message}\n"
+      "#{left.rjust(LEFT)}#{Time.now} | #{message}\n"
```

Comment:
Let's match the formatting Chef uses (via mixlib-log):

```
Time.now.iso8601()
```

Make sure you add a `require 'time'` so the `.iso8601()` method is available.

Example output:

```ruby
irb(main):001:0> require 'time'
=> true
irb(main):002:0> Time.now.iso8601()
=> "2016-04-27T14:31:11-04:00"
```